src/install-script/install-tortuga.sh

```diff
@@ -375,7 +375,8 @@ function install_puppetlabs_repo {
 
     # install puppetlabs-release
     pkgexists puppetlabs-release || {
-        rpm_install http://yum.puppetlabs.com/puppetlabs-release-el-${distmajversion}.noarch.rpm
+        rpm_install http://release-archives.puppet.com/yum/el/$(distmajversion}/products/x86_64/puppetlabs-release-22.0-2.noarch.rpm
```

Comment:
Something doesn't look right here... one of your parens around `distmajversion` is incorrect.

Reply:
You're absolutely right. I must have tested this only after pkgexists puppetlabs-release would succeed.